README.md

```diff
@@ -95,6 +95,13 @@ wireguard_postup:
 wireguard_postdown:
   - ...
 wireguard_save_config: "true"
+wireguard_unmanaged_peers:
+  client.example.com:
+    public_key: 5zsSBeZZ8P9pQaaJvY9RbELQulcwC5VBXaZ93egzOlI=
+    preshared_key: mfKkizs9J4eRFnyhaRvUKdFAxoGdDfcgMy4OhrbyjPE=
```

Comment:
Just a note. Maybe in the example use [`ansible-vault`](https://docs.ansible.com/ansible/latest/user_guide/vault.html) to provision the private key to promote good practices. :)